### ISSUE TITLE
Battery limit charging

### ISSUE THREAD

**Issue by amlboeton** [evanhuang117/T480s-hackintosh#1]
Does the battery charging limit working? I have T480s i5-8350u with Ventura 13.6.6 and the battery charging liimit is not working, even already set to 80% it's still charging beyond 80% all the time.

Previously on Windows through the Lenovo Vantage already set the threshold to 80% as well

**Comment by evanqhuang** [comment]
Yeah I think it should. It's been a while but I'm pretty sure it depends on the value set in Lenovo vantage which you said you did? I definitely remember it working but that was with Catalina.

**Comment by amlboeton** [comment]
> Yeah I think it should. It's been a while but I'm pretty sure it depends on the value set in Lenovo vantage which you said you did? I definitely remember it working but that was with Catalina.

previously on WIndows - Lenovo vantage I already set the charging limit to 80% but in hackintosh it's not working, it keep charging over 80% and it makes the battery cycle count changes where in windows it stays in 300 something but now it's reaching 600 over, is there any workout to make the charging limited to max 80% without add cycle count?
